_config.yml

```diff
@@ -15,7 +15,7 @@ exclude:
   - lang-ref
 
 # These allow the documentation to be updated with newer releases
-SYSTEMML_VERSION: 1.3.0-SNAPSHOT
+SYSTEMDS_VERSION: 2.0.0-SNAPSHOT
```

Comment:
I remember seeing `0.3.0-SNAPSHOT` during mvn build, not sure what the decision was.

Reply:
This is still up for debate, as discussed in the emails.
In my opinion the 2.0 seems the cleanest, since that wouldn't change previous releases.